test/unit/jwt-token-manager.test.js

```diff
@@ -175,19 +177,40 @@ describe('JWT Token Manager', () => {
     });
   });
 
+  describe('tokenNeedsRefresh', () => {
+    it('should return true if current time is past expire time', () => {
```

Comment:
I think the test description is a bit misleading as it talks about "expire time", but in practice this is checking the `refreshTime`.